### PR TITLE
Support new changes to WebXR API in Chrome M73

### LINKED ISSUE #291: gl.setCompatibleXRDevice, requestFrameOfReference is not a function
`session.setCompatibleXRDevice()` has been renamed to `session.makeXRCompatible()` in the latest Chrome Canary.

### PR DESCRIPTION
**Pull request by jsantell**
Support new changes to WebXR API in Chrome M73 with introduction of makeXRCompatible, requestReferenceSpace, and getViewerPose, as well as support the functions they've deprecated.

Tested in both Chrome Canary (73.0.3666.0), which uses the 3 above APIs, and Chrome Dev (73.0.3643.0), which uses the previous shape of all 3 features.

In the future, maybe we can use the [webxr-version-shim](https://github.com/immersive-web/webxr-samples/blob/spec-update/js/webxr-version-shim.js) being developed against the latest spec (which is only paritally implemented in Chrome, hence this issue), or a smaller bespoke abstraction once things stabilize, similar to #281.

Fixes #291

### CODE REVIEW

**Comment by jsantell** [comment]
We can also remove the older APIs and refer folks who want to support Chrome Dev, or non-latest Canary, to also include that shim, although in the mean time, supporting both could reduce questions or confusion on how to get AR working

File: src/three-components/ARRenderer.js [hunk 1]
```diff
@@ -198,7 +211,7 @@ export class ARRenderer extends EventDispatcher {
       this[$presentedScene].scaleModelToFitRoom();
     }
 
-    this[$frameOfReference] = null;
+    this[$refSpace] = null;
```
Comment (cdata) [inline src/three-components/ARRenderer.js]: Does changing `frameOfReference` to `refSpace` imply a corresponding change in the IDL?

**Comment by cdata** [comment]
Can we drop a comment note in the module basically echoing what you said in the PR description re: what are covering with the feature checks, and that we might switch to the shim later?

**Comment by mrdoob** [comment]
Chrome M63 is Chrome 73? 🤔

File: src/three-components/ARRenderer.js (hunk 1 above)
Comment (jsantell) [reply src/three-components/ARRenderer.js]: Yeah, this is an XRReferenceSpace rather than an XRFrameOfReference

**Comment by jsantell** [comment]
> Chrome M63 is Chrome 73?

oops

**Comment by jsantell** [comment]
* Summarized the compatibility issue a bit more in #293
* Updated comments s/M63/M73 and linked to #293 TODOs for the compatibility lines

**Pull request merged**